client/src/modules/templates/bhCardList.tmpl.html

```diff
@@ -0,0 +1,45 @@
+<div class="panel panel-default bh-card-list">
+  <div class="panel-heading">
+    <a href ng-click="$ctrl.toggleFilter()">
+      <span translate>TABLE.COLUMNS.FILTERING.LABEL</span>
+      <span class="caret" ng-class="{'rotate' : $ctrl.filterOptions.active}"></span>
+    </a>
+
+    <div class="pull-right">
+      <span class="label label-primary" style="margin-right: 5px;" ng-if="$ctrl.orderOptions.active">
+        <span translate>{{$ctrl.orderOptions.current.key}}</span>
+      </span>
+      <span uib-dropdown>
+        <a href uib-dropdown-toggle> <span translate>TABLE.COLUMNS.SORTING.LABEL</span> <span class="caret"></span></a>
+        <ul class="dropdown-menu-right textflow" uib-dropdown-menu>
+          <li ng-repeat="option in $ctrl.orderOptions.available">
+            <a href ng-click="$ctrl.setOrder(option)">
+              <span translate>{{ option.key }}</span>
+            </a>
+          </li>
+          <li role="seperator" class="divider"></li>
+          <li>
+            <a href ng-click="$ctrl.setOrder()">
+              <span class="fa fa-times" aria-hidden="true"></span>
+              <span translate> FORM.BUTTONS.CLEAR </span>
+            </a>
+          </li>
+        </ul>
+      </span>
+    </div>
+  </div>
+
+  <input
+    ng-model="$ctrl.filterOptions.value[$ctrl.currentFilterAttribute]"
+    ng-show="$ctrl.filterOptions.active"
+    class="form-control"
+    placeholder="{{'FORM.PLACEHOLDERS.FILTER_NAME' | translate }}"
+    style="border-radius: 0"/>
+</div>
+
+<div
+  ng-repeat="card in $ctrl.data
+    | filter : $ctrl.filterOptions.value : strict
+    | orderBy : 'label':$ctrl.orderOptions.current.reverse">
+  <ng-include src="$ctrl.template"></ng-include>
```

Comment:
Should we support pagination?  I know our Fiscal Year controller already has support for it via `uib-pagination`.  Do you think we could put it in here?

Reply:
I did notice pagination on the fiscal year controller. I think with all the modules in line to support this pattern none of them will have an overwhelming number of entities that would require both filtering and paginating. 

Definitely something to consider for future improvement though! The nice thing about setting up like this will be updating the Component to support pagination across all modules that implement this.